docs/features.md

```diff
@@ -2,11 +2,54 @@
 
 ## Nodes
 ### As a Node Operator, I want to install software from binary artifacts or a Docker image
-### As a Node Operator, I want to run software on Linux, MacOS, and in Docker
+
+ * A tarball containing latest version of rnode as a runnable .jar file is available at https://github.com/rchain/rchain/releases/
+ * A docker image containing latest version of rnode is available at https://hub.docker.com/r/rchain/rnode
+
+### As a Node Operator, I want to run software on Linux, macOS, and in Docker
+
+ * A .jar file containing latest version of rnode is runnable on Linux and macOS
+ * `docker pull rchain/rnode` followed by `docker run rchain/rnode` starts latest rnode version
+
 ### As a dApp Developer, I want to interface with the Rholang interpreter and evaluate smart contracts independently from the blockchain
 ### As a Node Operator, I want to have a default configuration and the ability to customize the configuration on the command line
+
+ * rnode reads in rnode.toml file at startup
+ * rnode.toml file settings may be passed also via command line, in which case command line ones take priority over the file
+ * Non-exhaustive list of configuration items:
+    * Bootstrap node address
+    * Bonds file path
+    * Wallets file path (in case of a bootstrap node)
+
 ### As a Node Operator, I want to monitor the performance, resource consumption, and status of my node
```

Comment:
@woky Any suggestions on how to best make sure the node is up and running, from the node operator point of view?  Some kind of a health check?  Perhaps something based on node metrics?

Reply:
@adaszko I'd check if the main port is listening. See also https://github.com/woky/rchain-misc/blob/master/ansible/testnet-setup/update-testnet.yml#L95. The availability of this port doesn't mean that Casper instance is available though. We discussed it last time and (at least) Dom agrees that node shouldn't open listening port unless it's *really ready*. In Docker with private network the `docker-proxy` process will listen on forwarded ports and I assume will close (RST) incoming connections until the forwarded port is listening inside network namespace. So successful TCP handshake is required to verify that port is open (i.e. not just list of listening sockets).

Reply:
https://github.com/rchain/rchain/pull/2051